examples/dataresidency/setRegion.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"github.com/sendgrid/rest"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+	"log"
+	"os"
+
+	"github.com/sendgrid/sendgrid-go"
+)
+
+var SAMPLE_EMAIL = "test@example.com"
+
+// SetDataResidency : Set region for sendgrid.
+func SetDataResidencyGlobal() {
+	message := buildHelloEmail()
+	request := buildSendgridObj("global")
```

Comment:
Can we call setDataResidency() method on the request object too? Right now, I guess the only way to set it is via the constructor, no?